src/pages/ItemsPage.vue

```diff
@@ -66,6 +73,56 @@ const filteredItems = computed(() => {
   return items.value
 })
 
+const sortTypeMap: ReadonlyArray<[string, string]> = [
+  ['0', '登録日時:古い順'],
+  ['1', '登録日時:新しい順'],
+  ['2', '名前順:昇順'],
+  ['3', '名前順:降順']
+]
+const typeOptions = sortTypeMap.map(([typeKey, typeName]) => ({
+  key: typeKey,
+  label: typeName
+}))
+
+const sortType = useDebouncedRef(
+  getFirstParam(route.query?.['sortType']) ?? '0'
+)
```

Comment:
```suggestion
const sortTypeOptions = [
  { key: 'CreatedAtAsc', label: '登録日時:古い順' },
  { key: 'CreatedAtDesc', label: '登録日時:新しい順' },
  { key: 'NameAsc', label: '名前順:昇順' },
  { key: 'NameDesc', label: '名前順:降順' }
] as const
type SortType = typeof opt[number]['key']

const sortType = useDebouncedRef<SortType>(
  getFirstParam(route.query?.['sort'] as SortType) ?? 'CreatedAtAsc'
)
```
typescriptのテクニック的なやつなので僕ならこうするを書いちゃうんですが、こんな感じにしましょう

理由は`const sortedItems = computed(() => { .... })`のなかでやってる`sortType.value === '0'`だとバグに気づきづらくなるからです
なんでバグに気づきづらくなるかというと
- `'0'`が何の順番を示しているのかわからない
- keyの名前を変えたいなと思った時に`sortTypeOptions`だけ変えて変更忘れに気づかないかもしれない(この変更を入れると`sortType.value === '間違ってるkey名'`みたいに変なものと比較してるとエラーになる)